examples/Cli/Help/Help.csproj

```
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net7.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>

```

```suggestion
    <Nullable>enable</Nullable>
    <ExampleName>Help</ExampleName>
    <ExampleDescription>Some description for help</ExampleDescription>
    <ExampleGroup>Cli</ExampleGroup>
    <ExampleVisible>false</ExampleVisible>
```

CLI examples are not really working for `dotnet example`, so we need to add the metadata and exclude it from appear in the listing.

---

Done